### DISCUSSION TITLE
I got a huge issue with my website! Why is it taking so long to process my website?

### DISCUSSION THREAD

**Discussion by Abigblueworld** [community/community#129132] [Programming Help]
### Select Topic Area

Question

### Feature Area

Issues

### Body

###Help needed!

So, on my website, it keeps not updating and I need help understanding this!

I've been though a lot of programming websites but this one I am so confused!

<b> How do I fix the slow live page time??? </b> 

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by floris-xlx** [comment]
can you attach some code for me to read, it is hard to judge when we dont know what framework or language you are in right now

share some code snippets thank you

**Reply by Abigblueworld** [reply]
I am working with HTML and CSS. Check abigblueworld.github.io
